anitya/lib/backends/stackage.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+"""
+ (c) 2014-2015 - Copyright Red Hat Inc
+ Authors:
+   Pierre-Yves Chibon <pingou@pingoured.fr>
```

Comment:
I guess my name can be removed from here until I actually contribute to this file ;-)